plugins/com.google.cloud.tools.eclipse.appengine.login/.classpath

```diff
@@ -4,10 +4,12 @@
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src"/>
 	<classpathentry kind="src" path="generated-sources"/>
+	<classpathentry exported="true" kind="lib" path="lib/com.google.cloud.tools.ide.login-0.0.0-SNAPSHOT.jar"/>
```

Comment:
Yeah, this is already a bundle. I couldn't find it in the Maven Central though. I guess someone (us?) need to publish it to the Maven Central first?